gulpfile.js

```diff
@@ -0,0 +1,140 @@
+const exec = require('child_process').exec;
+const fs = require('fs');
+const readline = require('readline');
+
+const cwsupload = require('chrome-webstore-upload');
+const eslint = require('gulp-eslint');
+const {google} = require('googleapis');
+const gulp = require('gulp');
+const jeditor = require('gulp-json-editor');
+const zip = require('gulp-zip');
+
+const CWS_ID = 'aojcceglbipehndciapjedoomockgagl';
+
+/**
+ * Checks the code for style errors.
+ */
+gulp.task('lint', () => {
+  return gulp.src(['src/**/*.js', 'gulpfile.js', '!node_modules/**'])
```

Comment:
Not sure, but if we're only globbing `src/ and gulpfile.js`, `gulp.src` will never match `'node_modules' right?

Reply:
Done